generators/entity/index.js

```diff
@@ -1499,7 +1576,7 @@ module.exports = EntityGenerator.extend({
             var entityNameSpinalCased = _s.dasherize(_s.decapitalize(this.name));
             var entityNamePluralizedAndSpinalCased = _s.dasherize(_s.decapitalize(pluralize(this.name)));
 
-            this.entityClass = _s.capitalize(this.name);
+            this.entityClass = this.entityName;
```

Comment:
the naming `this.entityName` is little confusing may be it should be `this.entityNameCapitalized` what do you think?